README.md

```diff
@@ -60,13 +60,13 @@ $ percy app:exec -- [appium test command]
 ## Configuration
 
 ```js
-percyScreenshot(driver, name[, {
+percyScreenshot(driver, name, {
```

Comment:
[ In node tells its optional  are we making it mandatory ? ]

Reply:
got it. i thought it was a bug. fixing it